ledger/src/blockstore.rs

```diff
@@ -136,10 +136,27 @@ impl std::fmt::Display for InsertDataShredError {
     }
 }
 
+/// A "complete data set" is a range of [`Shred`]s that combined in sequence carry a singled
+/// serialized [`Vec<Entry>`].
+///
+/// Services such as `solana_core::window_service::WindowService` for a TVU, and
+/// `solana_core::replay_service::ReplayStage` for a TPU, drive generation of these, via direct or
```

Comment:
nit: `drive generation of these via direct or indirect calls to` -> `piece together these sets by inserting shreds via`